app.js

```diff
@@ -17,6 +18,22 @@ function Project(name, link, description){
   this.about = description;
 
   this.addProject = function() {
-
+    // projects.push(this);
   }
 }
+
+var projects = [new Project('Bus Mall', 'https://rjb888.github.io/bus-mall/', 'Market research site for the Bus Mall'), new Project('HomeFit', 'https://famavott.github.io/Home-Workout-Generator/index.html','HomeFit is a web app I helped develop which was designed to allow a user to have a personalized workout generated for them based on their desired results.'), new Project('Salmon Cookies', 'https://rjb888.github.io/cookie-stand/','This was a site designed to help an entrapreneur manage his Salmon Cookie shops')];
```

Comment:
you'll want to move this to your data.js and create an array of project objects... like this

```
var projects = [
  {
    title: 'bus mall',
    description: 'this project was... interesting.',
  },
  {
    title: 'salmon cookies',
    description: 'also an interesting project',
  },
]
```